### ISSUE TITLE
dockerconfigjson secret type missing

### ISSUE THREAD

**Issue by DaviPtrs** [lensapp/lens#3715]
There's a mention on the [code](https://githubs.com/lensapp/lens/blob/3bec593bce22e54a8ef393dda0ef06b9600af58a/src/common/k8s-api/endpoints/secret.api.ts#L32) but you actually can't select it on Lens. I think the problem is located on [this piece of code](https://github.com/lensapp/lens/blob/HEAD/src/renderer/components/+config-secrets/add-secret-dialog.tsx#L79) where is missing secret types

**Comment by jim-docker** [comment]
Is this specifically what you are referring to:

![Screen Shot 2021-09-07 at 7 48 24 AM](https://user-images.githubusercontent.com/40840436/132339726-a5d3b13f-fafa-4ef8-8c26-64cd4c1a28c1.png)

**Comment by DaviPtrs** [comment]
Yup, I tried to add a new option to this 'SECRET TYPE' dropdown, but now I don't know how to convert the data to dockerconfigjson format. Seems like there's more code to do and I don't know Typescript enough
